snapcraft/internal/project_loader/_config.py

```diff
@@ -255,6 +256,20 @@ def _process_remote_parts(self, snapcraft_yaml):
         return snapcraft_yaml
 
 
+def _patchelf_install_required(project_options) -> bool:
+    is_xenial = False
+    with contextlib.suppress(os_release.errors.OsReleaseCodenameError):
+        release_codename = os_release.OsRelease().version_codename()
+        is_xenial = release_codename == 'xenial'
+
+    is_snap = common.is_snap()
+    is_environment = os.getenv('SNAPCRAFT_NO_PATCHELF')
+    is_arch_armhf = project_options.deb_arch == 'armhf'
+
+    return not (is_snap or is_environment or
+                (is_arch_armhf and is_xenial))
```

Comment:
We know for sure that releases later than xenial aren't broken?

Reply:
yes, look at rmadison or https://launchpad.net/ubuntu/+source/patchelf